guide/events/ingesting-usage.mdx

```diff
@@ -34,8 +34,7 @@ event is JSON with the following fields:
 
 ```json
 {
-  "external_customer_id": "__CUSTOMER_ID__", // (Required) Unique identifier of your customer performing the action
-  "external_subscription_id": "__SUBSCRIPTION_ID__" // (Optional) Required only in case of multiple plans for a single customer
+  "external_subscription_id": "__SUBSCRIPTION_ID__" // (Required) Required only in case of multiple plans for a single customer
```

Comment:
Should we change the description to something like "Unique identifier of your subscription"?

Reply:
Added to last commits (had a conflict with Mat's changes on this branch), but should be ok rn